invisible_cities/reco/peak_functions_test.py

```diff
@@ -597,3 +600,12 @@ def test_rebin_times_and_waveforms_times_are_consistent(t_and_wf, stride):
                                                   np.ones_like(wfs), stride)
 
     assert np.sum(rb_times) * stride == approx(np.sum(times))
+
+
+def test_rebin_times_and_waveforms_negative_bins(ICDATADIR):
+    pmap_file = os.path.join(ICDATADIR, 'pmaps_negative_bins.h5')
```

Comment:
maybe add a comment to explain what this test does? something like 'test that rebinning doesnt raise an error even when all the weights are negative". 
If in the future we introduce the masking of the negative signal we can reuse the test file to check if we indeed get rid of negative bins.